packages/dashboards/src/lib/configurator/services/configurator-heading.service.ts

```diff
@@ -0,0 +1,8 @@
+import { Injectable } from "@angular/core";
+import { BehaviorSubject } from "rxjs";
+
+@Injectable()
+
+export class ConfiguratorHeadingService {
```

Comment:
maybe a more generic name like `DimensionChangeNotificationService` would be good?

Reply:
Actually, maybe not since this is for a very specific problem.

Reply:
I had the same way of thinking and ended up with the very same conclusion :)